packages/expo-cli/src/commands/client/index.ts

```diff
@@ -370,23 +403,48 @@ export default function (program: Command) {
   program
     .command('client:install:android')
     .description('Install the Expo client for Android on a connected device or emulator')
+    .option(
+      '--latest',
+      'Ignore the local project version and install the latest available version of Expo client'
```

Comment:
```suggestion
      'Install the latest version of Expo client, ignoring the project\'s SDK version'
```
Maybe lead with the actionable info.